GitFyle.Core.Api/Models/Foundations/Contributions/Contribution.cs

```diff
@@ -17,6 +17,10 @@ public class Contribution
         public Guid ContributionTypeId { get; set; }
         public string ExternalId { get; set; }
         public string Title { get; set; }
+        public string CreatedBy { get; set; }
```

Comment:
Why are these added?

Reply:
@glhays  "The audit fields however is what we use when we create / update records on our side and we want to ensure that someone has not tampered with the model by enforcing the validation criteria for CreatedBy/UpdatedBy/CreatedWhen/UpdatedWhen" @cjdutoit due christo's comment  i just added them do we not need it?

Reply:
@cjdutoit Thoughts?

Reply:
Yes, the properties marked ExternalXXXXX is how it is created at the external repository.

CreatedBy/UpdatedBy/CreatedWhen/UpdatedWhen is our audit properties.  We will also validate them that they are the same on add and that the date is recent.

Reply:
@SlimAhmad Need new migration for these

Reply:
@SlimAhmad  don't do the migration for these.  I will fix the audit property names and on another PR and do the migration there.